d2renderers/d2svg/d2svg.go

```diff
@@ -125,7 +126,7 @@ func arrowheadDimensions(arrowhead d2target.Arrowhead, strokeWidth float64) (wid
 	return clippedStrokeWidth * widthMultiplier, clippedStrokeWidth * heightMultiplier
 }
 
-func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) string {
+func arrowheadMarker(isTarget bool, id string, bgColor string, connection d2target.Connection) string {
```

Comment:
does bgColor need to be passed in or will it only ever be the constant?

Reply:
Currently it would always be the constant